src/shapes/Object/AnimatableObject.ts

```diff
@@ -96,37 +56,21 @@ export abstract class AnimatableObject<
    */
   _animate<T extends number | TColorArg>(
     key: string,
-    to: T,
+    endValue: T,
     options: Partial<TAnimationOptions<T>> = {}
   ) {
     const path = key.split('.');
     const propIsColor = this.colorProperties.includes(path[path.length - 1]);
-    const currentValue = path.reduce((deep: any, key) => deep[key], this);
```

Comment:
This line handled the key path dot notation using `key.split('.')`

Reply:
i just moved it down, we need to execute it only if startValue is not passed.